salt/pillar/annotations.sls

```diff
@@ -16,21 +16,11 @@ annotations:
             client_secret: fake
         authority: null
         group: null
-    containerized: true
 
 elife:
     aws:
         access_key_id: AKIAFAKE
         secret_access_key: fake
-    php:
```

Comment:
This process is still configured on EC2 instances. It causes no harm, as we have 2 identical queue:watch, one running in a container and one on the host. It can be removed when we are sure the containerized one works.

Reply:
Safer to leave and add a TODO?

Reply:
It is still in place, this pillar does not affect EC2 instances but only Vagrant.